lib/org/chromium/webidl/Canonicalizer.js

```diff
@@ -112,6 +131,51 @@ foam.CLASS({
       },
     },
     {
+      documentation: `Determines if the two Enum definitions are joint sets. If
+          the sets are disjoint (i.e. there is a member that exists in one but
+          not the other), we emit an warning and null is placed as entry in
+          Canonical definitions. Otherwise, the superset is placed into
+          Canonical definitions.`,
+      name: 'enumCmp_',
+      code: function(left, right) {
+        // Work on clones of definitions.
+        var leftClone = left.clone();
```

Comment:
JS question: can you clone just the members? The rest of the object doesn't matter and isn't mutated, so cloning it isn't necessary.